pkgs/by-name/ds/dssat/package.nix

```diff
@@ -0,0 +1,71 @@
+{ stdenv
+, lib
+, cmake
+, glibc
+, gfortran
+, makeWrapper
+, fetchFromGitHub
+, fetchpatch
+, dos2unix
+, dataRepo ? fetchFromGitHub {
+    owner = "DSSAT";
+    repo = "dssat-csm-data";
+    rev = "v4.8.2.8";
+    hash = "sha256-hbSBKEvdSd1lfbemfp4Lk4/JcGMXGVjm1x7P7lmmuA0=";
+  }
+, ...
```

Comment:
This isn't a module. Is the `...` necessary?

```suggestion
```

Reply:
Thank you! Fixed.

I was using `nixpkgs-fmt` until now. Good to know. :+1: